.gitignore

```diff
@@ -2,6 +2,7 @@
 # dependencies
 /node_modules
 package-lock.json
+yarn.lock
```

Comment:
we might not want to upload the lock fie

Reply:
Do you mean we **might want** to upload it? 

Decided to remove it for now because it gave me problems when i wanted to install updated dependencies and lost some time finding out was due to the lock file.

Maybe we could consider re adding it later once the code is more stable?

Reply:
ahh no my bad i thought that we were adding it, we are in gitignore haha , yes i think that we should not upload it